app/controllers/orders_controller.rb

```diff
@@ -1,46 +1,49 @@
 class OrdersController < ApplicationController
 	before_action :authenticate_user!
 	before_action :redirect_if_traveler, only: %i[new]
+
 	def index
-		@orders = Order.all
+	  @orders = Order.where(user_id:current_user.id)
 	end
 
 	def new
-		if current_user.role=='sender'
-			@order = Order.new
-		end 
+	    if current_user.role=='sender'
+	      @order = Order.new
+	    end 
 	end
+
 	def create 
-		if current_user.role=='sender'
-			@order = Order.new(order_params)
-			@order.user_id = current_user.id
-			if @order.save 
-				redirect_to orders_path
+	  	if current_user.role=='sender'
+		 @order = Order.new(order_params)
+		 @order.user_id = current_user.id
+		   	if  @order.save 
+			 redirect_to orders_path
 			else
-				render :new 
-				flash.alert = "User not found."
+			 render :new 
+			 flash.alert = "User not found."
 			end 
 		end 
-	end
+	end 
 
 	def show
 		@order = Order.find(params[:id])
```

Comment:
use find_by instead of find @aleemchishti

Reply:
done